rotkehlchen/exchanges/bitstamp.py

```diff
@@ -115,11 +115,6 @@ def __init__(
         super().__init__('bitstamp', api_key, secret, database)
         self.base_uri = 'https://www.bitstamp.net/api'
         self.msg_aggregator = msg_aggregator
-        # NB: X-Auth-Signature, X-Auth-Nonce, X-Auth-Timestamp and Content-Type per request
-        self.session.headers.update({
```

Comment:
Why remove this from here? These headers will never need to be updated. Why update them in each api query?